app.rb

```diff
@@ -0,0 +1,123 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = 'microverse'
+  end
+
+  def start_library
+    puts 'Welcome to my School Library!'
+    until list_of_options
+      answer = gets.chomp.to_i
+      if answer == 7
+        puts 'You have selected exit! bye'
+        exit
+      end
+
+      user_input answer
+    end
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'Database is empty! Add a person.' if @people.empty?
+    @people.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
+
+  def create_person
+    print 'To create a student, press 1, to create a teacher, press 2 : '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. Try again'
+    end
+  end
+
+  def create_student
+    puts 'Create a new student'
+    print 'Enter student age: '
+    age = gets.chomp.to_i
+    print 'Enter name: '
+    name = gets.chomp
+    @people.push(Student.new(@classroom, age, name))
+    puts 'Student created successfully'
+  end

```

Comment:
- [ ] when creating the `Student` the application is supposed to ask if the student has parent permission or not, as you can see in the below screenshot this is not the case.
![Screenshot 2022-04-14 210842](https://user-images.githubusercontent.com/39240820/163460205-83402c9c-bda4-45c6-be1e-0d652ff43f38.png)